vendor/github.com/aquasecurity/libbpfgo/libbpfgo.go

```diff
@@ -357,6 +367,43 @@ func NewModuleFromFileArgs(args NewModuleArgs) (*Module, error) {
 	}, nil
 }
 
+func GetMapByName(name string) (*BPFMap, error) {
```

Comment:
This is a local update. The following [PR](https://github.com/aquasecurity/libbpfgo/pull/338) should integrate this function.